docs-ref-services/postgresql-mysql.md

```diff
@@ -5,18 +5,18 @@ keywords: Azure, Python, SDK, API, SQL, database, MySQL, PostgreSQL
 author: lisawong19
 ms.author: routlaw
 manager: douge
-ms.date: 07/19/2017
+ms.date: 10/02/2019
 ms.topic: article
 ms.prod: azure
 ms.technology: azure
 ms.devlang: python
 ---
 
-# Azure MySQL/PostgreSQL libraries for Python
+# Azure Database for MySQL/PostgreSQL libraries for Python
 
 ## MySQL
 
-Work with resources and data stored in [Azure MySQL Database](/azure/mysql/overview) from python with the MySQL manager and pyodbc.
+Work with resources and data stored in [Azure Database for MySQL](/azure/mysql/overview) from Python with the MySQL manager and pyodbc.
```

Comment:
Seems like this could be reworded...are "resources" stored in MySQL?

Something like:
This article/example/doc shows you how to xyz with a/b tech.

This sample will demonstrate how you can use Python and pyodbc to Azure Database for MySQL.

Reply:
I've added an intro sentence at the top of the article.